src/documents/timecard.md

```diff
@@ -10,15 +10,15 @@ After clicking save, the screen displays the recorded shift with the option to r
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2008.46.42.png)
 
-When editing the saved time, the function 'isFinishTimeOnNextDay' calculates whether the time is on the next day and the UI notifies the user with dynamic text:
+The user is notified if the finish time is on the next day using dynamic next:
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2009.26.14.png)
 
 The date input is conditionally rendered by clicking the checkbox. The dates are automatically inserted. If 'isFinishTimeOnNextDay' is true, the finish date is set to the next day. These can be manually changed: 
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2010.14.50.png)
 
-The user is returned to the the Timecard of the start date:
+The user remains on the start date of the Timecard:
```

Comment:
The user remains day they were currently viewing, referencing the start date is misleading